schemas/index.d.ts

```diff
@@ -60,6 +60,68 @@ export interface Outcome {
   metrics: string[];
   slug: string;
 }
+export interface NimbusExperiment {
+  schemaVersion: string;
+  slug: string;
+  id: string;
+  appName: string;
+  appId: string;
+  channel: string;
+  userFacingName: string;
+  userFacingDescription: string;
+  isEnrollmentPaused: boolean;
+  isRollout?: boolean;
+  bucketConfig: BucketConfig;
+  outcomes?: Outcome1[];
+  featureIds: string[];
+  branches: (SingleFeatureBranch | MultiFeatureDesktopBranch | MultiFeatureMobileBranch)[];
+  targeting?: string;
+  startDate?: string;
+  enrollmentEndDate?: string;
+  endDate?: string;
+  proposedDuration?: number;
+  proposedEnrollment?: number;
+  referenceBranch?: string;
+  featureValidationOptOut?: boolean;
+  localizations?: {
+    [k: string]: unknown;
+  };
+  locales?: string[];
+}
+export interface BucketConfig {
+  randomizationUnit: string;
+  namespace: string;
+  start: number;
+  count: number;
+  total: number;
+}
+export interface Outcome1 {
```

Comment:
Uh oh -- name conflict. It added a `1` to the end because `jetstream` has its own `Outcome`.

Reply:
I'll just rename it

Reply:
Alright, that's fine for now but we may want to think about an approach going forward. I don't like how this seemed like a silent error that would be very easy to miss and confusing. Maybe we can at least error/warn on name conflicts via CI as opposed to the TS conversion just renaming without telling us. Maybe a strict linting that doesn't allow TS types to end with a number?

Reply:
No I think the strategy here is to write tests that hit the ts generated schemas.  If the renaming had broken the interface the tests would have failed.

Reply:
Oh yea that's better, I'll add a ticket.

Reply:
Already filed ;) https://github.com/mozilla/experimenter/issues/9368